components/BaseContestCard.tsx

```diff
@@ -27,22 +28,22 @@ export default function BaseContestCard({
           key={contest.id}
           className='relative rounded-lg overflow-hidden shadow-m h-full w-full'
           style={{
-            backgroundImage: `url(${pictureCover})`,
+            backgroundImage: `url(${awsEnv + pictureCover})`,
             backgroundSize: 'cover',
             height: '15em',
           }}
         >
           <div className='absolute inset-0 bg-black opacity-40 rounded-lg'></div>
-          <div className='absolute inset-0 flex flex-col justify-start items-start'>
+          <div className='absolute inset-0 flex flex-col justify-start items-start h-fix'>
             {children[0]}
-            <h3 className='text-white font-medium text-lg mt-10 ml-2'>
+            <h3 className='text-inter text-[#F3F3F4] font-medium text-sm/[18px] mt-10 ml-2 opacity-50'>
```

Comment:
Consider adding frequently used colors/ class combinations as new classes in tailwind
https://tailwindcss.com/docs/adding-custom-styles